jsx/src/components/ServerDashboard/ServerDashboard.jsx

```diff
@@ -41,7 +41,7 @@ const ServerDashboard = (props) => {
   let user_data = useSelector((state) => state.user_data);
   const user_page = useSelector((state) => state.user_page);
 
-  const { setOffset, offset, setLimit, handleLimit, limit, setPagination } =
+  const { offset, setLimit, handleLimit, limit, setPagination } =
     usePaginationParams();
```

Comment:
usePaginationParams still defines and returns `setOffset`, and I expected this assignment to run into issues due to that - but even if it works due to name matching of some kind, maybe `setOffset` should be removed from `usePaginationParams`?

```
  return {
    offset,
    setOffset,
    limit,
    setLimit,
    handleLimit,
    setPagination,
  };
```

Reply:
Ah, I saw below...

```
    return updateUsers({
      offset: offset,
      limit,
```

So I presume that `{variable1, key2: variable2}` becomes `{variable1: variable1-value, key2: variable2-value}` and that makes it possible to name-match in the assignment etc.

---

Okay, so __its just a question about cleanup of `setOffset` or not remaining__